tests/integration/migrator/compiled_account_contract_tx_v0.py

```diff
@@ -0,0 +1,2 @@
+# OpenZeppelin Account implementation v0.3.1 copied from Starknet Devnet v0.3.0
```

Comment:
I think this type of fixtures should be shared across tests. Consider moving it to `tests/data`.

Reply:
I guarantee you that it's not going to be used anywhere else.